dependencies.tsv

```diff
@@ -7,6 +7,7 @@ github.com/armon/go-metrics	git	93f237eba9b0602f3e73710416558854a81d9337	2017-01
 github.com/beorn7/perks	git	3ac7bf7a47d159a033b107610db8a1b6575507a4	2016-02-29T21:34:45Z
 github.com/bmizerany/pat	git	c068ca2f0aacee5ac3681d68e4d0a003b7d1fd2c	2016-02-17T10:32:42Z
 github.com/boltdb/bolt	git	e9cf4fae01b5a8ff89d0ec6b32f0d9c9f79aefdd	2017-01-31T19:20:18Z
+github.com/cloud-green/monitoring	git	666e3beca3cb4f6b5c8f176ba80daf2b3adbd84e	2017-11-27T13:22:20Z
```

Comment:
don't need this one I don't think; I've seen it before as a test only transitive dependency so is not needed here

Reply:
I've confirmed this can be removed

Reply:
do we need this?